sources/engine/Stride.Rendering/Rendering/Shape.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing.Printing;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Stride.Core;
+
+namespace Stride.Rendering
+{
+    [DataContract]
+    // Represet one instance of blend shape and control points
+    public class Shape
+    {
+        public string Name { get; set; }
+
+        public string ChannelName { get; set; }
+
+        public string Shapeindex { get; set; }
+
+
+
+        //Index Of control points in shape
+        public int[] Indices { get; set; }
+
+        Vector4[] _positions;
+        public Vector4[] Positions
+        {
+            get
+            {
+                return _positions;
+            }
+            set
+            {
+                if (value == null) { _positions = null; return; }
+                
+                Position = new Vec4[value.Length];
+                for (int i = 0; i < value.Length; i++)
+                {
+                    Position[i] = value[i] * .01f;
```

Comment:
What's this multiplier about ? Is that because of how fbx scales things ? Is there a way to extract that from the fbx ?

Reply:
Yes. I too was also not sure about this. The weight data in .FBX ranges 0-100. I normalized to 0-1 as more intuitive ranging to developers. but can remove the multiplier and keep it as it is. Please let me know what you think!

Reply:
Ah, okay, that makes sense. Can you move that to the importer code, that way if we have other formats that support blend shapes in the future they won't have this fbx-specific scaler ?

Reply:
Ah, okay, that makes sense. Can you move that to the importer code, that way if we have other formats that support blend shapes in the future they won't have this fbx-specific scaler ?

Reply:
Done!